specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/config/proxy.when.topic.with.messages.yaml

```diff
@@ -13,25 +13,26 @@
 # specific language governing permissions and limitations under the License.
 #
 
-#TODO: this should be the final config
```

Comment:
Some adjustments needed here for `publish` and `subscribe` syntax under `when` condition.